src/refract/util.es6

```diff
@@ -0,0 +1,45 @@
+/*
+ * Returns `true` if there is a `class` attribute list and `name` is present
+ * in the list.
+ */
+function hasClass(element, name) {
+  return element.meta && element.meta.class &&
+         element.meta.class.indexOf &&
```

Comment:
Looks like the recommended way to check for an array is instead to use `Array.isArray(value)`, so I'll do that instead.